Lib/test/test_email/test_headerregistry.py

```diff
@@ -1628,7 +1629,7 @@ def test_address_display_names(self):
                     'Lôrem ipsum dôlôr sit amet, cônsectetuer adipiscing. '
                     'Suspendisse pôtenti. Aliquam nibh. Suspendisse pôtenti.',
                     '=?utf-8?q?L=C3=B4rem_ipsum_d=C3=B4l=C3=B4r_sit_amet=2C_c'
-                    '=C3=B4nsectetuer?=\n =?utf-8?q?adipiscing=2E_Suspendisse'
+                    '=C3=B4nsectetuer?=\n =?utf-8?q?_adipiscing=2E_Suspendisse'
```

Comment:
Note, the data in the unittest was buggy.  If you run the original output through `decode_header()` you'll find that it is missing the space between `cônsectetuer` and `adipiscing`